server/app/user.py

```diff
@@ -0,0 +1,30 @@
+"""Module that includes functionality to work with user`s data."""
+
+import logging
+
+from asyncpg import exceptions
+
+
+LOG = logging.getLogger(__name__)
+
+
+CREATE_USER = """
+    INSERT INTO "USER" (telegram_id)
+         VALUES ($1);
+"""
+
+
+class User:
+    """Model that provides methods to work with user`s data."""
+
+    def __init__(self, postgres=None, redis=None):
+        """Initialize user instance with required clients."""
+        self._postgres = postgres
+        self._redis = redis
+
+    async def create_user(self, telegram_id):
+        """Create new user in database."""
+        try:
+            return await self._postgres.execute(CREATE_USER, telegram_id)
```

Comment:
I think, initially, we can use a user's name from telegram so we can use it in the future before the user adds his monobank API key

Reply:
I'm not sure if it will be useful. Because using our app is beginning after initializing monobank.
But OK, I'll add.